misc/specs/atrium-specs-common/src/main/kotlin/ch/tutteli/atrium/specs/integration/IterableFeatureAssertionsSpec.kt

```diff
@@ -0,0 +1,77 @@
+package ch.tutteli.atrium.specs.integration
+
+import ch.tutteli.atrium.api.fluent.en_GB.*
+import ch.tutteli.atrium.api.verbs.internal.expect
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.specs.*
+import ch.tutteli.atrium.translations.DescriptionIterableAssertion
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.Suite
+import kotlin.math.exp
+
+abstract class IterableFeatureAssertionsSpec(
+    minFeature: Feature0<Iterable<Int>,Int>,
```

Comment:
as the return type is different, cant use Fun0 here and hence using Feature0. but the subclass of this does not compile